tests/auth-middleware.test.ts

```
@@ -0,0 +1,260 @@
+ *   - res.locals.accountId from JWT payload when JWT auth is used (VAL-AUTH-ID-003)
+ *   - requireAccount rejects when res.locals.accountId is undefined (VAL-AUTH-ID-004)
+ */
+
+import { afterAll, beforeAll, describe, expect, test } from "bun:test";
+import express, { type Response } from "express";
+import { authOrAgentApiKeyAuth } from "@/middleware/agentAuth";
+import { requireAccount } from "@/middleware/auth";
+import { jsonMiddleware } from "@/middleware/json";
+import { pinoMiddleware } from "@/middleware/pino";
+import { getEffectiveOwnerId } from "@/utils/auth-helpers";
+import { createJwtToken } from "@/utils/jwt";
```

ESLint

[error] Replace `createJwtToken·}·from·"@/utils/jwt` with `ADMIN_ACCOUNT_ID·}·from·"@/utils/constants`

(prettier/prettier)